ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -164,42 +344,49 @@ typeof define === 'function' && define.amd ? define(['exports'], factory) :
 
     // Render data into three.js objects
     const materials = [];
-    for (let i = 0; i < structure.visualization.length-1; i++) {
+    for (let i = 0; i < structure.visualization.length - 1; i++) {
       materials.push(new THREE.LineBasicMaterial(
           {color: Colors.random()} ));
     }
     const objects = {};
     for (let i = 0; i < layerCoordinates.length; i++) {
       objects[structure.visualization[i].id] = [];
-      const lineSegments = [];
+      const lineSegmentsGeometry = [];
       for (const line of layerCoordinates[i]) {
         const linePoints = [];
-        for (let j = 0; j < line.length; j+=ONE_POINT) {
-          linePoints.push( new THREE.Vector3( line[j], line[j+1],
-              line[j+2]+FLOOR_HEIGHT*structureIndex ) );
+        for (let j = 0; j < line.length; j += ONE_POINT) {
+          linePoints.push( new THREE.Vector3( line[j], line[j + 1],
+              line[j + 2] + FLOOR_HEIGHT * structureIndex ) );
         }
         const geometry = new THREE.BufferGeometry().setFromPoints(
             linePoints );
         if (line.length === TWO_POINTS) {
           // group geometries for performance reason
-          lineSegments.push( geometry );
+          lineSegmentsGeometry.push( geometry );
         } else {
-          objects[structure.visualization[i].id].push( new THREE.LineLoop(
-              geometry, materials[i] ) );
+          const lineLoop = new THREE.LineLoop( geometry, materials[i] );
+          lineLoop.visible = false;
+          scene.add( lineLoop );
+          objects[structure.visualization[i].id].push( lineLoop );
         }
       }
-      if (lineSegments.length > 0) {
+      if (lineSegmentsGeometry.length > 0) {
         const geometries = THREE.BufferGeometryUtils.mergeBufferGeometries(
-            lineSegments );
-        objects[structure.visualization[i].id].push( new THREE.LineSegments(
-            geometries, materials[i] ) );
+            lineSegmentsGeometry );
+        const lineSegments = new THREE.LineSegments( geometries, materials[i] );
+        lineSegments.visible = false;
+        scene.add( lineSegments );
+        objects[structure.visualization[i].id].push( lineSegments );
       }
     }
     return objects;
   }
 
-  exports.renderLayer = renderLayer;
-  exports.unpackStructure = unpackStructure;
+  exports.createSidebar = createSidebar;
+  exports.generateScene = generateScene;
```

Comment:
if `renderTopIBRStructure` and `createSidebar` are the only ones being used currently, we probably shouldn't need to export the other functions